js/iprange.go

```diff
@@ -0,0 +1,192 @@
+package js
+
+import (
+	"encoding/binary"
+	"fmt"
+	"math/rand"
+	"net"
+	"strconv"
+	"strings"
+)
+
+type SelectMode uint
+
+const (
+	LoopIncSelectIP SelectMode = iota // increase one by one, no IP is leaking
+	RandomSelectIP                    // random
+	ModLenSelectIP                    // like LoopIncSelect, may leak IP(s) if id is not continous
+)
```

Comment:
This will need to have better user-facing names, even if they are only two see [compatibilityMode](https://github.com/loadimpact/k6/blob/master/lib/runtime_options.go#L30-L40) for how it's done in other places

Reply:
any suggestion for the names?